pkg/apis/core/v1/conversion.go

```diff
@@ -555,3 +563,113 @@ func Convert_core_PersistentVolumeSpec_To_v1_PersistentVolumeSpec(in *core.Persi
 func Convert_v1_PersistentVolumeSpec_To_core_PersistentVolumeSpec(in *v1.PersistentVolumeSpec, out *core.PersistentVolumeSpec, s conversion.Scope) error {
 	return autoConvert_v1_PersistentVolumeSpec_To_core_PersistentVolumeSpec(in, out, s)
 }
+
+// HandleSeccompVersionSkew implements the version skew behavior described in:
+// https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/20190717-seccomp-ga.md#version-skew-strategy
+func HandleSeccompVersionSkew(in *core.Pod, out *v1.Pod) error {
```

Comment:
@pjbgf what about the security context inside of the pods containers? Do we have to update/sync them as well?

Reply:
That's correct, we need to extend the same logic for containers when the annotation `SeccompContainerAnnotationKeyPrefix` + container name is being used.

Reply:
Added the handling for the containers, too.